.github/workflows/bump-python-package.yml

```diff
@@ -17,7 +17,7 @@ on:
 
 jobs:
   bump-python-package:
-    runs-on: ubuntu-latest
```

Comment:
What's wrong with using `ubuntu-latest` everywhere? Is the idea that we fully pin/freeze our CI environments?